### PR TITLE
Fix #3581 Merged topic_lessons_story_summary into single xml file.

### LINKED ISSUE #3581: Merge topic_lessons_story_summary.xml into single xml file
Currently there are 4 versions of `topic_lessons_story_summary.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by puranjayK**
## Explanation
Fixes #3581 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

Before Changes             |  After Changes
:-------------------------:|:-------------------------:
  | 
 | 
 | 
 |

### CODE REVIEW

**Comment by puranjayK** [comment]
@rt4914 PTAL

**Comment by rt4914** [comment]
@puranjayK Thanks for the PR. As mentioned in the issue description add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

**Comment by puranjayK** [comment]
I have updated the comment
@rt4914 PTAL

File: app/src/main/res/values-land/dimens.xml [hunk 1]
```diff
@@ -328,4 +328,11 @@
   <dimen name="ongoing_topic_list_fragment_padding_top">28dp</dimen>
   <dimen name="ongoing_topic_list_fragment_padding_end">56dp</dimen>
   <dimen name="ongoing_topic_list_fragment_padding_bottom">88dp</dimen>
+
+  <!--  Topic Lessons Story Summary-->
```
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml]: Update to
`<!-- Topic Lessons Story Summary -->`

Basically one space before and after the text. Currently you have 2 space before and 0 space after.

Ditto for other files.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -518,4 +518,11 @@
   <dimen name="ongoing_topic_list_fragment_padding_top">20dp</dimen>
   <dimen name="ongoing_topic_list_fragment_padding_end">20dp</dimen>
   <dimen name="ongoing_topic_list_fragment_padding_bottom">144dp</dimen>
+
+<!--  Topic Lessons Story Summary-->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: The alignment of comment is incorrect. It should start from where you below line is starting.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@puranjayK Please add tablet mobile/portrait before/after screenshots too.


**Note**:
Whenever you remove a tablet xml file you will need to perform following steps to see the fresh changes.
1. Uninstall the previous app.
2. `Build` -> `Rebuild` the app.
3. Install the app again.

**Comment by puranjayK** [comment]
@rt4914
I'm getting the wrong layout in the tablet mode. 
The lesson story summary cardview had "layout_gravity=centre" attribute earlier in the XML file. Can you please tell what are the margins in the tablet mode.

File: app/src/main/res/values-land/dimens.xml (hunk 1 above)
Comment (puranjayK) [reply app/src/main/res/values-land/dimens.xml]: Done

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (puranjayK) [reply app/src/main/res/values/dimens.xml]: Done

**Comment by rt4914** [comment]
@puranjayK For tablet the content width is fixed instead of margin. Have a look https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/a98b7a3c-46c4-49c6-af77-14a1ff5b7b03/

The content width is fixed to "528dp"

**Comment by puranjayK** [comment]
@rt4914 
How should I keep the card view in centre then in tablet mode?

**Comment by rt4914** [comment]
> @rt4914 How should I keep the card view in centre then in tablet mode?

@puranjayK Either introduce constraints or use layout_gravity. Or else instead of merging it to single file merging it to 2/3 files which can work best for the app.

**Comment by puranjayK** [comment]
> > @rt4914 How should I keep the card view in centre then in tablet mode?
> 
> @puranjayK Either introduce constraints or use layout_gravity. Or else instead of merging it to single file merging it to 2/3 files which can work best for the app.

Merged the files into 2 XML files.
Updated the comment with screenshots

**Comment by puranjayK** [comment]
> > > @rt4914 How should I keep the card view in centre then in tablet mode?
> > 
> > 
> > @puranjayK Either introduce constraints or use layout_gravity. Or else instead of merging it to single file merging it to 2/3 files which can work best for the app.
> 
> Merged the files into 2 XML files. Updated the comment with screenshots

@rt4914 PTAL

**Review by rt4914 — CHANGES_REQUESTED** [review]
@puranjayK PTAL

The `story_name_chapter_count_container` TextView has different `layout_marginStart` for tablet landscape and portrait.

Also, please merge with latest develop too.

**Comment by puranjayK** [comment]
> @puranjayK PTAL
> 
> The `story_name_chapter_count_container` TextView has different `layout_marginStart` for tablet landscape and portrait.
> 
> Also, please merge with latest develop too.

@rt4914 I will make the necessary changes. Till then is it possible if you could add "hacktoberfest-accepted" label to the PR?

**Comment by rt4914** [comment]
@puranjayK I am not sure about that. I will talk to @BenHenning about this. As far as I know hacktoberfest has already ended.

**Comment by BenHenning** [comment]
Hi @puranjayK. Sorry for not following up earlier. For Oppia Android, we only consider PRs accepted for Hacktoberfest when they've been merged since that ensures that we can maintain a consistent quality bar across all contributions to the project.

Please let me know if you have any further questions.

**Comment by rt4914** [comment]
@puranjayK This PR cannot be labelled for `Accepted`. I do understand that part of this was delayed PR review but we need multiple PRs for hacktoberfest and also that too merged before 31st Oct.
https://hacktoberfest.digitalocean.com/faq

**Comment by rt4914** [comment]
Closing this PR because of inactivity.

**Pull request closed without merging**